src/ui/server/shared-validation/alpha-characters-only/index.ts

```diff
@@ -11,20 +10,7 @@ import generateValidationErrors from '../../helpers/validation';
  * @param {Object} Validation errors
  * @returns {Object | Boolean} Validation errors
  */
-const alphaCharactersOnlyValidation = (fieldValue: string, fieldId: string, errorMessage: string, errors: object) => {
-  const joiString = Joi.string();
-
-  const regex = REGEX.ALPHA_CHARACTERS_AND_SPACE;
-
-  const schema = () => joiString.regex(regex).required();
-
-  const validation = schema().validate(fieldValue);
-
-  if (validation.error) {
-    return generateValidationErrors(fieldId, errorMessage, errors);
-  }
-
-  return false;
-};
+const alphaCharactersOnlyValidation = (fieldValue: string, fieldId: string, errorMessage: string, errors: object) =>
+  regexValidation(fieldValue, fieldId, REGEX.ALPHA_CHARACTERS_AND_SPACE, errorMessage, errors);
```

Comment:
Could we update `numberHyphenSpacesOnlyValidation` to use `regexValidation? Then all instances of `joiString.regex(...` are aligned